src/interface/wallet.cpp

```diff
@@ -127,6 +127,10 @@ class WalletImpl : public Wallet
 public:
     WalletImpl(CWallet& wallet) : m_wallet(wallet) {}
 
+    void markDirty() override
+    {
+        m_wallet.MarkDirty();
```

Comment:
what exactly does MarkDirty do?

Reply:
Same as before, marking the wallet dirty :D Basically just forcing to recalculate cached balances of the wallet/wallet transactions.